### PR TITLE
Contact list Created

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Create a ContactListItem, add a database and map it in the index.js to a list.
Please check it out. Thank you

### CODE REVIEW

**Comment by TmWgnr** [comment]
ola maria, your code looks good to me.  
only thing i am not sure if ur project is connected to the vercel. can´t open ur vercel page.

**Comment by MariaRiosNavarro** [comment]
[Deploy](https://mobile-simulation-prop-lobw2rsod-mariariosnavarro.vercel.app/)

File: lib/db.js [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+      "🥳",
+      "Denk daran, dass ich dich unterstütze",
+      "Ich bin so dankbar",
+      "Ich bewundere deine Entschlossenheit und deinen Mut",
+      "Fühl dich gedrückt!",
+    ],
+  },
+];
+//   {
+//     id: "6",
```
Comment (FelineHuhn) [inline lib/db.js]: You maybe should remove the commented-out objects, or?

File: components/ContactListItem/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { StyledHR } from "../StyledHR";
+
+const StyledContainer = styled.div`
+  display: flex;
+  align-items: center;
+  /* justify-content: space-between; */
```
Comment (FelineHuhn) [inline components/ContactListItem/index.js]: You should remove this line, since it is out-commented, or?

File: lib/db.js (hunk 1 above)
Comment (MariaRiosNavarro) [reply lib/db.js]: i do it

File: components/ContactListItem/index.js (hunk 2 above)
Comment (MariaRiosNavarro) [reply components/ContactListItem/index.js]: i do it

**Pull request merged**